test/application/parser/parser-v1.test.js

```diff
@@ -55,6 +63,10 @@ describe('Parser test suite', function () {
       expect(parsedCertificate.recipientFullName).toEqual(fullNameAssertion);
     });
 
+    it('should set recordLink of the certificate object', () => {
```

Comment:
I think we have a convention around using `function` rather than arrow functions (to prevent scope leakage), but maybe it was unpublicized and just in my head. But we should follow it.

Reply:
We haven't talked about it indeed but it's a good thing that this thought is getting out of your head. I'll update those.